### ISSUE TITLE
Constraint on DMS

### ISSUE THREAD

**Issue by LunaAsefaw** [iwmihq/miniSASS#630]
If I am not mistaken, the range for DMS Latitude is 0 to 90 and DMS Longitude is 0 to 180, right @NyakudyaA ?

Currently, the form accepts any values
![image](https://github.com/kartoza/miniSASS/assets/90245715/3befbe3e-777d-4a9f-95bc-098b462cc6ad)

**Comment by sokolic** [comment]
Correct, the DMS latitude range is 0-90, the longitude range is 0-180. Minutes and Seconds have a range of 0-60.

**Comment by LunaAsefaw** [comment]
Thanks for the clarification @sokolic

**Comment by sokolic** [comment]
Currently the decimal degrees allows inputs to four decimal places while the DMS input allows input of seconds to 2 decimal places. In order to make them equivalent you would need the Decimal Degrees to allow inputs to 6 decimal places. Also, given that 1 second represents about 30 metres distance on the ground, I would suggest the following: For Decimal Degrees allows inputs to 6 decimal places; For DMS allow only whole numbers for the Degrees, four decimal places for the Minutes and two decimal places for the Seconds. Configuring the inputs like this would allow users to input either (1) Decimal Degrees, or (2) Degrees and Decimal Minutes, or (3) Degrees, Minutes and Decimal Seconds, while allowing for GPSs that capture points with a positional accuracy of about 30 cm.
